tests/js/server/dump/dump-db-properties-teardown-cluster-single.js

```diff
@@ -0,0 +1,43 @@
+/*jshint globalstrict:false, strict:false */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief teardown for dump/reload tests
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2019 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Christoph Uhde
+/// @author Copyright 2019, ArangoDB GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+const db = require("@arangodb").db;
+
+function tearDown(name, minReplicationFactor, replicationFactor, sharding){
+  db._dropDatabase(name + "Src");
+  db._dropDatabase(name + "Dst");
+}
+
+const doIt = tearDown;
+
+doIt("UnitTestsDump", 2, 3, "");
```

Comment:
this whole script can be simplified to just
```
const db = require("@arangodb").db;
db._dropDatabase("UnitTestsDumpSrc");
db._dropDatabase("UnitTestsDumpDst");
return {
  status: true
};
```
which would be much more legible.